signup/tool/src/java/org/sakaiproject/signup/tool/jsf/SignupUIBaseBean.java

```diff
@@ -738,11 +754,13 @@ public String getAllAttendeesEmailAddressesFormatted() {
 		Set<String> emails = new HashSet<String>();
 		
 		StringBuilder sb = new StringBuilder();
-		for (TimeslotWrapper tsWrapper : timeslotWrappers) {
-			for(AttendeeWrapper atWrapper : tsWrapper.getAttendeeWrappers()) {
-				String email = atWrapper.getAttendeeEmail();
-				if(StringUtils.isNotBlank(email)){
-					emails.add(email);
+		if (timeslotWrappers!=null){

```

Comment:
Moreover, the signup tool in general has a lot of a brittle code - making something non-brittle within it I would say is only a nice-to-have.